Windows: Fails to start in memory DB

On Windows with this config:

```properties
quarkus.datasource.db-kind=sqlite
quarkus.datasource.jdbc.url=jdbc:sqlite::memory:
```

I get this stack:

```
2025-02-08 13:01:54,088 ERROR [io.qua.run.Application] (Quarkus Main Thread) Failed to start application: java.lang.RuntimeException: Failed to start quarkus
        at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
        at io.quarkus.runtime.Application.start(Application.java:101)
        at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:121)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:77)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:48)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:137)
        at io.quarkus.runner.GeneratedMain.main(Unknown Source)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
        at java.base/java.lang.reflect.Method.invoke(Method.java:580)
        at io.quarkus.runner.bootstrap.StartupActionImpl$1.run(StartupActionImpl.java:116)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.IllegalStateException: Error starting Liquibase
        at io.quarkus.liquibase.runtime.LiquibaseRecorder.doStartActions(LiquibaseRecorder.java:111)
        at io.quarkus.runner.recorded.LiquibaseProcessor$startLiquibase1744275855.deploy_0(Unknown Source)
        at io.quarkus.runner.recorded.LiquibaseProcessor$startLiquibase1744275855.deploy(Unknown Source)
        ... 11 more
Caused by: java.lang.IllegalStateException: java.nio.file.InvalidPathException: Illegal char <:> at index 0: :memory:
        at io.quarkus.liquibase.LiquibaseFactory.createLiquibase(LiquibaseFactory.java:138)
        at io.quarkus.liquibase.LiquibaseFactory_1d48OwssPZoQytD2p8kGRAP54yY_Synthetic_ClientProxy.createLiquibase(Unknown Source)
        at io.quarkus.liquibase.runtime.LiquibaseRecorder.doStartActions(LiquibaseRecorder.java:88)
        ... 13 more
Caused by: java.nio.file.InvalidPathException: Illegal char <:> at index 0: :memory:
        at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:204)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:175)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
        at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
        at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:231)
        at java.base/java.nio.file.Path.of(Path.java:148)
        at io.roastedroot.sqlite4j.core.WasmDB._open(WasmDB.java:279)
        at io.roastedroot.sqlite4j.core.DB.open(DB.java:216)
        at io.roastedroot.sqlite4j.SQLiteConnection.open(SQLiteConnection.java:298)
        at io.roastedroot.sqlite4j.SQLiteConnection.<init>(SQLiteConnection.java:75)
        at io.roastedroot.sqlite4j.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:28)
        at io.roastedroot.sqlite4j.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:19)
        at io.roastedroot.sqlite4j.JDBC.createConnection(JDBC.java:118)
        at io.roastedroot.sqlite4j.JDBC.connect(JDBC.java:91)
        at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:225)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:580)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:561)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
        at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
```

--- a/pom.xml
+++ b/pom.xml
@@ -23,7 +23,7 @@
   </scm>
   <properties>
     <quarkus.version>3.18.2</quarkus.version>
-    <sqlite4j.version>3.48.0.0</sqlite4j.version>
+    <sqlite4j.version>3.48.0.1</sqlite4j.version>
     <icu4j.version>76.1</icu4j.version>
   </properties>
   <dependencyManagement>

